eng/Publishing.props

```diff
@@ -31,12 +31,15 @@
   </ItemGroup>
 
   <Target Name="_PublishInstallersAndChecksums">
-    <!-- 
-      This target is defined in eng/targets/Packaging.targets and included in every C# and F# project.
-      We use Microsoft.AspNetCore.DeveloperCertificates.XPlat because it is a nonshipping package, and we need a non-stable version string to use as our publish location.
-      If Microsoft.AspNetCore.DeveloperCertificates.XPlat ever becomes a shipping package, this logic will break, so be careful
+    <!--
+      This target is defined in eng/targets/Packaging.targets and Npm.Common.targets and included in every C#, F#,
+      and npm project. We use signalr.npmproj because it is a nonshipping package, we need a non-stable version
```

Comment:
>We use signalr.npmproj because it is a nonshipping package

It is a shipping package o.O

Reply:
> It is a shipping package o.O

Maybe in 3.1.x. Here, it won't get stable versions and lands in the artifacts/packages/Release/NonShipping folder. Is that a separate bug❔ 

![image](https://user-images.githubusercontent.com/6431421/89725443-80721900-d9c4-11ea-9532-1eb65f979612.png)

Reply:
First off, that's the java package (which should also be shipping so that's a different issue).

![image](https://user-images.githubusercontent.com/7574801/89725520-389fc180-d9c5-11ea-9ca8-88fdc30098bd.png)

Reply:
Yeah, I mixed up the extensions. Using signalr.client.java.Tests.javaproj would also have been wrong❔ Please file an issue to ship that.

Guess I'll have to find a non-C#, non-F# project that's neither shipping nor packed. Will anything go sideways if I use src/SignalR/clients/ts/common/common.npmproj (after marking it explicitly not to ship) or &hellip;/FunctionalTests/SignalR.Npm.FunctionalTests.npmproj❔ Any preference❔

Reply:
Well FuntionalTests will never have a possibility of shipping, so that's my preference

Reply:
Done. I also double-checked a local binary log and confirmed SignalR.Npm.FunctionalTests.npmproj has `$(IsShipping)` and `$(IsShippingPackage)` equal to `false`.